vendor/github.com/pires/go-proxyproto/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
this is licensed under Apache, but `Conn#readHeader()` indicates it is using code from github.com/armon/go-proxyproto which is MIT-licensed... I'm unsure about that